test/TinyRAM/Spec/EncodeInstruction.hs

```diff
@@ -16,7 +16,7 @@ import           TinyRAM.Types.Word                (Word (..))
 encodeInstruction :: RegisterCount -> Instruction -> (Word, Word)
 encodeInstruction rc i =
   ( Word $ fromIntegral (i ^. #opcode . #unOpcode)
-       .|. aBit
+       .|. aBit `shift` 5
```

Comment:
Why should it be necessary to bit shift here? The value `aBit` is already bit shifted.